calico-policies/private-network-isolation/calico-v3/us-south/allow-egress-pods-private.yaml

```diff
@@ -116,10 +116,13 @@ spec:
       - 166.9.51.54/32
       - 166.9.51.74/32
       - 166.9.51.106/32
+      - 166.9.51.107/32
```

Comment:
These private IPs throughout the PR aren't correct, they should be the ones that resolve the three individual hostnames that the workers use:
c118-1-1.private.us-south.containers.cloud.ibm.com: 166.9.48.175
c118-2-1.private.us-south.containers.cloud.ibm.com: 166.9.51.104
c118-3-1.private.us-south.containers.cloud.ibm.com: 166.9.58.170